hoodie-bench/src/main/java/com/uber/hoodie/integrationsuite/sink/writer/FileSinkWriter.java

```diff
@@ -0,0 +1,37 @@
+/*
+ *  Copyright (c) 2019 Uber Technologies, Inc. (hoodie-dev-group@uber.com)
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *           http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ *
+ */
+
+package com.uber.hoodie.integrationsuite.sink.writer;
+
+import java.io.IOException;
+
+/**
+ * This extends {@link SinkWriter} and allows to have a file system based output sink
+ */
+public interface FileSinkWriter<I> extends SinkWriter<I> {
+
+  /**
+   * Open the path/file to write the data to
+   */
+  void open(String path) throws IOException;
+
+  /**
+   * Start a new file sink writer and close the current one
```

Comment:
Looks like the implementation does not auto-close. Either the javadoc needs to be fixed or the implementation needs to change.

Reply:
fixed